extensions/2.0/Vendor/EXT_mesh_features/README.md

```diff
@@ -703,3 +703,4 @@ Composite|A glTF containing a 3D mesh (house), a point cloud (tree), and instanc
   * Refactored the property texture schema so it is now a glTF `textureInfo` object.
   * Each property texture now contains only a single texture
   * Property textures are now assumed to be in linear space, and must use nearest or linear filtering
+  * Added `id` to schema
```

Comment:
Maybe something like this to be clearer that the change is referring to the metadata schema, not just the JSON schema?

```suggestion
  * Added a `schema.id` property
```